tests/xorg-test-run.sh

```diff
@@ -1,13 +1,25 @@
-#!/bin/sh
+#!/bin/bash
 
 # Test that Xorg can load the compiled modules
 
 # X server to run
 if [ -z "$XORG" ]; then
-    if [ -x /usr/lib/xorg/Xorg ]; then
-        # Don't use the Ubuntu wrapped server
-        XORG=/usr/lib/xorg/Xorg
-    else
+    xorg_paths=(
+        /usr/local/libexec/Xorg
+        /usr/libexec/Xorg
+        /usr/lib/xorg/Xorg
+        /usr/lib/Xorg
+        /usr/bin/Xorg
+    )
+
+    for path in "${xorg_paths[@]}"; do
+        if [ -x "$path" ]; then
+            XORG=$path
+            break
+        fi
+    done
+
+    if [ -z "$XORG" ]; then
```

Comment:
Since we're not using the positional parameters, I'd suggest the following for sh:-

```sh
set -- \
    /usr/local/libexec/Xorg \
    /usr/libexec/Xorg \
    /usr/lib/xorg/Xorg \
    /usr/lib/Xorg \
    /usr/bin/Xorg

    for path in "$@"; do
    <etc>
```